index.js

```diff
@@ -595,43 +595,41 @@ function saveHtmlFile(html) {
 }
 
 function animateMove(originJQueryElement, destinationJQueryElement) {
+  destinationJQueryElement.css("visibility", "hidden");
```

Comment:
looks like a lot of changes, but actually just removing the outer-wrapping `setTimeout` of 1000 ms